github/repos.go

```diff
@@ -305,6 +305,28 @@ func (s *RepositoriesService) Get(ctx context.Context, owner, repo string) (*Rep
 	return repository, resp, nil
 }
 
+// Get the contents of a repository's code of conduct.
```

Comment:
`golint` will complain here, saying the doc string should begin with `// GetCodeOfConduct ...`, so you should write:

```Go
// GetCodeOfConduct gets the contents of a repository's code of conduct.
```